hbase-server/src/test/java/org/apache/hadoop/hbase/master/assignment/TestSplitTableRegionProcedure.java

```diff
@@ -191,15 +156,13 @@ public void testRollbackForSplitTableRegionWithReplica() throws Exception {
     final TableName tableName = TableName.valueOf(name.getMethodName());
     final ProcedureExecutor<MasterProcedureEnv> procExec = getMasterProcedureExecutor();
 
-    RegionServerHostingReplicaSlowOpenCopro.slowDownReplicaOpen = true;
+    RegionServerHostingReplicaSlowOpenCoprocessor.slowDownReplicaOpen = true;
     RegionInfo[] regions =
       MasterProcedureTestingUtility.createTable(procExec, tableName, null, columnFamilyName1);
 
-    try {
-      HBaseTestingUtil.setReplicas(UTIL.getAdmin(), tableName, 2);
-    } catch (IOException ioe) {
-
-    }
+    TableDescriptor td = TableDescriptorBuilder.newBuilder(UTIL.getAdmin().getDescriptor(tableName))
+      .setRegionReplication(2).build();
+    procExec.submitProcedure(new ModifyTableProcedure(procExec.getEnvironment(), td));
```

Comment:
We don't need to mess with `hbase.client.sync.wait.timeout.msec` if we bypass the Admin client